Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -120,7 +120,11 @@ public func verifySnapshot<Value, Format>(
 
       guard !recording, fileManager.fileExists(atPath: snapshotFileUrl.path) else {
         try snapshotting.diffing.toData(diffable).write(to: snapshotFileUrl)
-        return "Recorded snapshot: …\n\n\"\(snapshotFileUrl.path)\""
+        return """
+        An existing reference was not found on disk. Automatically recorded snapshot: …
+
+        "\(snapshotFileUrl.path)"
```

Comment:
Could further improve this error with more information if you have ideas...

Reply:
spitballing:

> An existing reference was not found on disk. Automatically recorded snapshot: …
>
> "\\(snapshotFileUrl.path)"
>
> Run \\(testName) again to test against the newly recorded snapshot.